src/execution/operator/persistent/physical_insert.cpp

```diff
@@ -14,11 +14,14 @@ namespace duckdb {
 //===--------------------------------------------------------------------===//
 class InsertGlobalState : public GlobalSinkState {
 public:
-	InsertGlobalState() : insert_count(0) {
+	InsertGlobalState() : insert_count(0), returned_chunk_count(0) {
+		return_chunk_collection = ChunkCollection();
```

Comment:
This is not required - in C++ the empty constructor will be called by default for all non-trivial data types. Only numerics & pointers need to be explicitly instantiated.